docs/teams/shared_secrets.md

```diff
@@ -0,0 +1,25 @@
+# Shared secrets management
+
+## Overview
+
+This feature allows your team to share Livebook secrets among team members in an easy way.
+
+Just add a secret to your organization hub, and Livebook Teams will synchronize it with the Livebook of every member of your organization.
+
+![](assets/add_shared_secret.png)
+
+It also synchronizes updates and deletions of secrets.
+
+## How it works
+
+Here's a video showing how that feature works.
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/GENSmArO1AI?si=pvMJt1Ihr5UBCkPP" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment:
Should we also say you can create such secrets directly from a notebook?

Reply:
Done.